Source/SmallBasic.Editor/Components/Layout/MainLayout.scss

```diff
@@ -26,7 +26,7 @@ main-layout {
             align-items: center;
 
             logo {
-                @include set-background-image("Logo.png", 224px, 80px);
+                @include set-background-image("logo.svg", 224px, 80px);
```

Comment:
Thanks for looking into this!
Is the old logo used anywhere else? can we delete it now?

Reply:
Doesn't look like it's used elsewhere. Deleted!